vector/src/main/res/layout/view_url_preview.xml

```diff
@@ -1,85 +1,79 @@
-                android:layout_marginEnd="8dp"
-                android:layout_marginTop="6dp"
-                android:ellipsize="end"
-                android:maxLines="1"
-                android:singleLine="true"
-                android:textColor="?vctr_content_secondary"
-                tools:text="BBC News" />
+            android:adjustViewBounds="true"
+            android:importantForAccessibility="no"
+            android:scaleType="fitXY"
+            tools:src="@tools:sample/backgrounds/scenic" />
 
-            <TextView
-                android:id="@+id/url_preview_title"
-                style="@style/Widget.Vector.TextView.Body.Medium"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:layout_marginStart="8dp"
-                android:layout_marginEnd="8dp"
-                android:layout_marginTop="4dp"
-                android:ellipsize="end"
-                android:maxLines="2"
-                android:textColor="?vctr_content_primary"
-                android:textStyle="bold"
-                tools:text="Jo Malone denounces her former brand's John Boyega decision" />
+        <TextView
+            android:id="@+id/url_preview_site"
+            style="@style/Widget.Vector.TextView.Caption"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="8dp"
+            android:layout_marginTop="6dp"
+            android:layout_marginEnd="8dp"
+            android:ellipsize="end"
+            android:maxLines="1"
+            android:singleLine="true"
+            android:textColor="?vctr_content_secondary"
+            tools:text="BBC News" />
 
-            <TextView
-                android:id="@+id/url_preview_description"
-                style="@style/Widget.Vector.TextView.Body"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:layout_marginStart="8dp"
-                android:layout_marginEnd="8dp"
-                android:layout_marginTop="4dp"
-                android:layout_marginBottom="8dp"
-                android:ellipsize="end"
-                android:maxLines="2"
-                android:textColor="?vctr_content_secondary"
-                tools:text="The British perfumer says removing actor John Boyega from his own advert was “utterly despicable”." />
+        <TextView
+            android:id="@+id/url_preview_title"
+            style="@style/Widget.Vector.TextView.Body.Medium"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="8dp"
+            android:layout_marginTop="4dp"
+            android:layout_marginEnd="@dimen/layout_touch_size"
+            android:ellipsize="end"
+            android:maxLines="2"
+            android:paddingEnd="8dp"
```

Comment:
Maybe we do not need paddingEnd anymore

Reply:
will remove :+1: didn't realise the close icon already had intrinsic padding 
![2021-12-02T10:24:09,757711187+00:00](https://user-images.githubusercontent.com/1848238/144403909-8b0f4c46-5d24-4670-b5bc-1768d06726fb.png)

Reply:
https://github.com/vector-im/element-android/pull/4610/commits/26ba63d7f1d9fc821d5aa4b5ea050ecfd319a9e2